lib/CoreBot.js

```diff
@@ -679,6 +701,12 @@ function Botkit(configuration) {
 
     botkit.spawn = function(config, cb) {
         var worker = new this.worker(this, config);
+
+        if (config.token_wit) {
```

Comment:
I'm not quite sure this is the correct place to instantiate witbot. I believe these lines change the architecture in a bad way.